### PR TITLE
new viewset feature: patch-all

### PR DESCRIPTION
**Pull request by aleontiev**

### CODE REVIEW

File: dynamic_rest/viewsets.py [hunk 1]
```diff
@@ -418,36 +441,150 @@ def _bulk_update(self, data, partial=False):
...
+        fields = serializer.get_all_fields()
+        validated = {}
+        for name, value in six.iteritems(data):
+            field = fields.get(name, None)
+            if field is None:
+                raise ValidationError(
+                    'Unknown field: "%s"' % name
+                )
+            source = field.source or name
+            if source == '*':
```
Comment (ryochiji) [inline dynamic_rest/viewsets.py]: Do we also need to check for `read_only` or would that happen upstream somewhere?

File: tests/viewsets.py [hunk 2]
```diff
@@ -151,6 +151,7 @@ class DogViewSet(DynamicModelViewSet):
     model = Dog
     serializer_class = DogSerializer
     queryset = Dog.objects.all()
+    ENABLE_PATCH_ALL = True
```
Comment (ryochiji) [inline tests/viewsets.py]: Should we disable by default?

Comment (ryochiji) [reply tests/viewsets.py]: nm

File: dynamic_rest/viewsets.py [hunk 3]
```diff
@@ -418,36 +441,150 @@ def _bulk_update(self, data, partial=False):
...
+                'fur': 'gold'
+            }
+        """  # noqa
         if self.ENABLE_BULK_UPDATE:
-            partial = 'partial' in kwargs
-            bulk_payload = self._get_bulk_payload(request)
-            if bulk_payload:
-                return self._bulk_update(bulk_payload, partial)
+            patch_all = self.get_request_patch_all()
+            if (
```
Comment (ryochiji) [inline dynamic_rest/viewsets.py]: style: line wrapping probably not necessary here

Comment (aleontiev) [reply dynamic_rest/viewsets.py]: :+1:, I think this initially had longer names and one extra clause so it was too long to fit on one line

File: dynamic_rest/viewsets.py (hunk 1 above)
Comment (aleontiev) [reply dynamic_rest/viewsets.py]: Good catch, should check for it here

**Pull request merged**